sql/hive/src/main/antlr3/org/apache/spark/sql/parser/SparkSqlParser.g

```diff
@@ -637,7 +637,7 @@ import org.apache.hadoop.hive.conf.HiveConf;
   // counter to generate unique union aliases
   private int aliasCounter;
   private String generateUnionAlias() {
-    return "_u" + (++aliasCounter);
+    return "u_" + (++aliasCounter);
```

Comment:
This change is because Hive lexer doesn't allow identifiers starting with underscore.

(All other changes in this file are caused by removing training spaces.)